plugins/win-capture/graphics-hook/obs-vulkan64.json

```diff
@@ -0,0 +1,49 @@
+{
+    "file_format_version": "1.1.2",
+    "layer": {
+        "name": "VK_LAYER_OBS_HOOK",
+        "type": "GLOBAL",
+        "library_path": ".\\graphics-hook64.dll",
+        "api_version": "1.1.114",
```

Comment:
Should probably build against whatever the latest Vulkan SDK is, and bump this number.

Reply:
We don't actually link to vulkan-1.dll directly, it's all indirect through getprocaddress calls, so the version number here probably isn't super critical.